ches/emailHandler.ts

```diff
@@ -93,3 +98,45 @@ export const sendEditRequestEmails = async (
     console.error('ERROR SENDING EDIT REQUEST EMAIL');
   }
 };
+
+export const sendDeleteRequestEmails = async (product: PrivateCloudRequestedProjectWithContacts, comment: string) => {
+  const email = render(DeleteRequestTemplate({ product }), { pretty: true });
+
+  try {
+    await sendEmail({
+      body: email,
+      to: [product.projectOwner.email, product.primaryTechnicalLead.email, product.secondaryTechnicalLead?.email],
+      subject: `${product.name} deletion request has been rejected`,
+    });
+  } catch (error) {
+    console.error('ERROR SENDING REQUEST REJECTION EMAIL');
+  }
+};
+
+export const sendDeleteRequestApprovalEmails = async (product: PrivateCloudRequestedProjectWithContacts) => {
+  const email = render(DeleteApprovalTemplate({ product }), { pretty: true });
+
+  try {
+    await sendEmail({
+      body: email,
+      to: [product.projectOwner.email, product.primaryTechnicalLead.email, product.secondaryTechnicalLead?.email],
+      subject: `${product.name} deletion request has been approved`,
+    });
+  } catch (error) {
+    console.error('ERROR SENDING REQUEST REJECTION EMAIL');
```

Comment:
The error message for the send delete emails need to be updated

Reply:
fixed